### ISSUE TITLE
npm install problem with lodash

### ISSUE THREAD

**Issue by xeBuz** [fractile81/cordova-phaser-ts-jed#1]
Hi

I tried to run `npm install` but I got this error (in 2 different computers)

```bash
Running "typings:install" (typings) task
error!
TypingsError: Unable to read typings for "lodash". You should check the entry paths in "lodash.d.ts" are up to date
  at /Users/jesusroldan/Dev/TestCordovaPhaserTs/node_modules/typings-core/dist/lib/compile.js:217:35
  at process._tickCallback (internal/process/next_tick.js:103:7)

Warning: Task "typings:install" failed. Use --force to continue.

Aborted due to warnings.

npm WARN grunt-express@1.4.1 requires a peer of grunt@~0.4.0 but none was installed.
npm WARN grunt-po2json@0.3.0 requires a peer of grunt@~0.4.1 but none was installed.
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/7.2.1/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v7.2.1
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! testcordovaphaserts@1.0.0 install: `grunt clean:postinstall typings:install`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the testcordovaphaserts@1.0.0 install script 'grunt clean:postinstall typings:install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the testcordovaphaserts package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt clean:postinstall typings:install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs testcordovaphaserts
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls testcordovaphaserts
npm ERR! There is likely additional logging output above.
```

I tried also:
```
typings install lodash
```
But it didn't works

**Comment by fractile81** [comment]
Replace the contents of your `typings.json` file with the following:

```
{
  "globalDependencies": {
    "cordova": "registry:dt/cordova#0.0.0+20160819210047",
    "phaser": "github:photonstorm/phaser/v2/typescript/typings.json"
  },
  "dependencies": {
    "lodash": "registry:npm/lodash#4.0.0+20161015015725"
  }
}
```

Run `typings install` after doing so and things should "just work".

I'll get this into the repo later tonight.

**Comment by fractile81** [comment]
This has been fixed in release v0.3.1
